### DISCUSSION TITLE
Setting Envmap Stops sRGB Output Encoding For 1 Render

### DISCUSSION THREAD

**Discussion by josheppinette** [pmndrs/postprocessing#474] [Q&A]
This is super weird, but when I change the `scene.envmap` to non-null, the  next `composer.render()` will not use sRGB output encoding on the scene's background. If I immediately call `composer.render()` again, and every subsequent time - it will properly use the sRGB output encoding for the background.

Note, it is only the scene background which is not being encoded properly.

**Comment by vanruesc** [comment]
> when I change the scene.envmap

Do you mean `Scene.background`?

> the next composer.render() will not use sRGB output encoding on the scene's background

That sounds like a bug, but if it only affects the scene background, then the root cause will probably not be in `postprocessing` but rather in `three`. We'd need a minimal reproduction of the issue to find out when and why this happens.
